x/docs/track.rb

```diff
@@ -20,7 +20,7 @@ def initialize(data, repository)
       attr_reader :repository, :data
 
       def value(topic)
-        if data[topic].empty?
+        if data[topic].blank?
```

Comment:
This will handle both `nil` and `""` cases. `data[topic]` will be `nil` if we don't serve track specific `launch` copy.